conductr_cli/bndl_main.py

```diff
@@ -127,18 +123,18 @@ def build_parser():
 
     endpoint_args = parser.add_argument_group('endpoints')
     endpoint_args.add_argument('-e', '--endpoint',
-                               help='Endpoints to add to bundle\n'
+                               help='Endpoints that are added to the bundle\n'
                                     'If specified, existing endpoints are removed\n'
                                     'Example: bndl --endpoint web --component web --bind-protocol http '
                                     '--service-name web --acl http:/subpath',
-                               metavar='ENDPOINT [OPTS]',
+                               metavar='ENDPOINT',
                                dest='endpoint_dicts',
                                default=[],
                                action=EndpointAction)
     endpoint_args.add_argument('--component',
                                help='Component to which an endpoint should be added\n'
                                     'Used in conjunction with the --endpoint option\n'
-                                    'Required when specifying the --endpoint option',
+                                    'When absent, the endpoints of the first component are replaced',
```

Comment:
The implementation is a bit different than the help text suggestions - it only autodetects if there is one component, it doesn't just use the first.

One idea would be to replace "When absent, the endpoints of the first component are replaced" with "Required when a bundle has more than one component"